build.sbt

```diff
@@ -76,7 +76,8 @@ lazy val pekkoPlugin = project
     addSbtPlugin(
       // When updating the sbt-paradox version,
       // remember to also update project/plugins.sbt
-      "com.lightbend.paradox" % "sbt-paradox" % "0.10.3"),
+      ("com.lightbend.paradox" % "sbt-paradox" % "0.10.3").exclude("com.typesafe.sbt", "sbt-web")),
```

Comment:
Why `0.10.3` here? Shouldn't that be either downgraded to 0.9.2 or upgraded to 0.10.6?

Reply:
This is a mistake, can you fix it?

Reply:
Pushed